tests/nnstreamer_datarepo/unittest_datareposink.cc

```diff
@@ -376,6 +465,8 @@ remove_test_file (void)
   g_remove ("video.raw");
   g_remove ("mnist.json");
   g_remove ("mnist.data");
+  g_remove ("flexible.data");
+  g_remove ("flexible.json");
```

Comment:
Is there any reason to remove the file after the all test is finished rather than the test where the file was created is finished?

Reply:
Ok, Thank you.